app/controllers/approval_requests_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ApprovalRequestsController < ApplicationController
+  def index
+    requests = current_user.approval_requests.collect do |record|
```

Comment:
It will generates a lot of sql queries(as many as user has approval reqeust). You can select them all by one query:
```ruby
approval_requests_ids = current_user.approval_requests.pluck(:vacation_request_id).compact
requests = VacationRequest.find approval_requests_ids
```